### DISCUSSION TITLE
cprotectinline could be merged with cprotect

### DISCUSSION THREAD

**Discussion by blefloch** [user202729/TeXlib#2] [General]
(This would be more appropriate as an email, but I didn't find a usable email address.)

Does it make sense to include \cprotectinline as a new feature of the cprotect package?  (I haven't looked in any detail at what it does.)  If you think it would make sense, then I can upload cprotect to a new github repo (after converting the history from mercurial to git) and you can open a pull request.

**Comment by user202729** [comment]
But I'm not the author of that package...? (ah you are? Well it does work. I have some other internal code to preserve synctex info for it however, which depends on a bunch of other so-far-unuploaded packages, so it could be a bit difficult though. Also... I don't know but you probably won't like my coding convention...?)

What about simply mentioning "related packages" in the documentation?
